administrator-manual/en/accounts.rst

```diff
@@ -270,92 +270,112 @@ Joining an Active Directory domain has the following pre-requisite:
   with the privilege of joining a computer to the domain. Remember that the
-  default *administrator* account could be disabled!
+  default *administrator* account could be disabled! Press :guilabel:`Check` again.
 
-.. index::
-     pair: service; account
+* If the credentials are valid complete the procedure by pressing :guilabel:`Next`.
 
-.. _ad-dedicated-service-account:
-
-.. warning::    Some additional modules, like *Nextcloud*, *WebTop*, *Roundcube*, *Ejabberd*
-                require read-only access to AD LDAP services. To be fully operational they
-                require an additional account to perform simple LDAP binds.
-                
-                Create a **dedicated user account** in AD, and set a complex *non-expiring*
-                password for it.
-                
-Once |product| has successfully joined AD, specify the **dedicated user account**
-credentials in :guilabel:`Accounts provider > Authentication credentials for LDAP applications`.
+Some applications require an additional configuration step. See also :ref:`dedicated-service-account`.
 
 .. _bind-remote-ldap-section:
 
 Bind to a remote LDAP server
 ----------------------------
 
-To configure a remote LDAP accounts provider, go to page :guilabel:`Accounts
-provider > LDAP > Bind remotely`.
+To configure a remote LDAP accounts provider, go to page :guilabel:`Users & Groups
+> Configure the account provider > LDAP > Bind remote LDAP`.
 
-Type the LDAP server IP address in the field :guilabel:`Host name or IP`. If
+Type the LDAP server IP address in the field :guilabel:`Hostname or IP`. If
 the LDAP service runs on a non-standard TCP port, specify it in :guilabel:`TCP
-port`.
+port`. Press the :guilabel:`Check` button to proceed.
 
 Then an LDAP *rootDSE* query is sent to the specified host and a form is filled
 with returned data.  Check the values are correct then press the
-:guilabel:`Save` button to confirm.
+:guilabel:`Check` button again.
 
-If the LDAP server requires authentication, fill in the fields under
-:guilabel:`Authenticated bind`. Enable either ``ldaps://`` or STARTTLS to 
-encrypt the connection.
+If the LDAP server requires authentication, set :guilabel:`Bind Type` to :guilabel:`Authenticated`.
+Set either ``ldaps://`` in :guilabel:`Service URI` or enable :guilabel:`StartTLS` to encrypt the connection.
 
 .. tip::
 
     If the remote LDAP server is also a |product| installation and
     it is in the LAN (green) network, select :guilabel:`Anonymous bind`
-    
+
+Some applications require an additional configuration step. See also :ref:`dedicated-service-account`.
+
+
+.. index::
+     pair: service; account
+
+.. _dedicated-service-account:
+
+.. _ad-dedicated-service-account:
+
+LDAP account for additional applications
+----------------------------------------
+
+Some additional applications, like *Nextcloud*, *WebTop*, *Roundcube*, *Ejabberd*,
+require read-only access to LDAP services. To be fully operational they
```

Comment:
```suggestion
require read-only access to remote LDAP services. To be fully operational they
```

Reply:
Why _remote_? That's true despite of the account provider thing...

Reply:
Because for local AD and LDAP everything is auto-configured.

Reply:
Ok, let's say that explicitly, because it might be not clear that the whole section applies to remote providers only (described in the above sections).

Reply:
Please review again